content/language-agnostic/encyclopedia/what-is-a-parameter-and-an-argument?.md

```diff
@@ -0,0 +1,28 @@
+<!--
+---
+Subjects:
+  - "What is a parameter and an argument?"
```

Comment:
From the template: "# Please first look to use Subjects that are already listed at author.codecademy.com/browse/categories"